packages/server-core/src/user/identity-provider/identity-provider.test.ts

```diff
@@ -317,4 +317,100 @@ describe('identity-provider.test', () => {
       }
     )
   })
+
+  it('should not allow search without the required scope - multiple queries', async () => {
```

Comment:
Please add a test case when the user is admin then then he should be able to make the queries.